frontend/src/pages/ui/SignupSection.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Grid, TextField } from "@material-ui/core";
+import React from "react";
+import { CONTACT_TAB_LINK, LinkWithStickyParams } from "../../utils/urlutils";
+import styles from "./SignupSection.module.scss";
+
+export default function SignupSection() {
+  return (
+    <Grid
+      container
+      direction="column"
+      justify="center"
+      className={styles.NewsEmailSignup}
+    >
+      <Grid item>
+        <p>
+          Please{" "}
+          <LinkWithStickyParams to={CONTACT_TAB_LINK}>
+            contact us
+          </LinkWithStickyParams>{" "}
+          with any questions or concerns.
+        </p>
+        <p>
+          For more information about health equity, please sign up for our
+          Satcher Health Leadership Institute newsletter.
+        </p>
+      </Grid>
+      <Grid item container justify="center" alignItems="center">
+        <form
+          action="https://satcherinstitute.us11.list-manage.com/subscribe?u=6a52e908d61b03e0bbbd4e790&id=3ec1ba23cd&"
```

Comment:
Should this be a constant?

Reply:
Yes, good call; I'll move to url utils. We should implement some automated tests that runs through all of our external urls and confirms a successful http status.

Reply:
Yeah Larry was talking about that as a general principal. All links should always work.

Reply:
I'll look into this. I honestly have such a limited background in testing and would love the experience of setting some up

Reply:
Maybe we can use a tool like whats described here in our linting step? https://dev.to/modern-web/introducing-check-html-links-no-more-bad-links-1jdg

Reply:
Looks like someone made a github action for it which would be perfect. Look under the heading "Usage Github Action
"